test/configuration_test.rb

```diff
@@ -605,25 +605,47 @@ class FileSystem
     end
   end
 
-  describe '#default_format' do
+  describe '#default_request_format' do
+    describe "when not previously set" do
+      it 'returns :html' do
+        @configuration.default_request_format.must_equal :html
+      end
+    end
+
+    describe "when set" do
+      before do
+        @configuration.default_request_format :json
+      end
+
+      it 'returns the value' do
+        @configuration.default_request_format.must_equal :json
+      end
+    end
+
+    it 'raises an error if the given format cannot be coerced into symbol' do
+      -> { @configuration.default_request_format(23) }.must_raise TypeError
+    end
+  end
+
+  describe '#default_response_format' do
     describe "when not previously set" do
       it 'returns nil' do
-        @configuration.default_format.must_equal :html
+        @configuration.default_response_format.must_equal nil
```

Comment:
@theocodes HTML must be honored. Lotus::Controller is designed for HTTP endpoints, so it uses a generic content type (`application/octet-stream`, Rack default).

Lotus full stack app are designed to be web apps, where HTML must be the default.